src/common/net.cc

```diff
@@ -337,6 +370,41 @@ namespace Pistache
 
     void Address::init(const std::string& addr)
     {
+        // Handle unix domain addresses separately.
+        if (isUnixDomain(addr))
+        {
+            struct sockaddr_un unAddr;
+            unAddr.sun_family = AF_UNIX;
+
+            // See unix(7) manual page; distinguish among unnamed, abstract,
+            // and pathname socket addresses.
+            auto size = std ::min(addr.size(), sizeof unAddr.sun_path);
+            if (size == 0)
+            {
+                addrLen_ = sizeof unAddr.sun_family;
+            }
+            else if (addr[0] == '\0')
+            {
+                addrLen_ = static_cast<socklen_t>(
+                    sizeof unAddr.sun_family + size);
+                ::memcpy(unAddr.sun_path, addr.data(), size);
```

Comment:
If you really want to specify the namespace of `memcpy()` and `strncpy()` then use `std::memcpy`, since technically speaking the `cstdlib` header only provides functions in the `std` namespace

Reply:
Fixed.